src/solc_0.7/catalyst/AssetAttributesRegistry.sol

```diff
@@ -23,7 +23,8 @@ contract AssetAttributesRegistry is WithMinter, WithUpgrader, IAssetAttributesRe
 
     struct Record {
         uint16 catalystId; // start at 1
-        uint16[MAX_NUM_GEMS] gemIds; // start at 1 test compression ?
+        // @todo start at 1 test compression ?
```

Comment:
@wighawag Can you elaborate on this comment?

Reply:
Ha, hmm, I am not sure what I was trying to say :( 
I think we can safely ignore though